tasks/util.py

```diff
@@ -1587,6 +1587,13 @@ def run(self):
         after = time.time()
         LOGGER.info('time: %s', after - before)
 
+        LOGGER.info('analyzing the table %s', self.output().table)
+        session = current_session()
+        session.execute('ANALYZE {table}'.format(table=self.output().table))
```

Comment:
Depending in the size of the table this could take some time to finish

Reply:
Maybe we should move the analyze inside the `check_null_columns` because is a dependency for that module to avoid removing by mistake or something

Reply:
I tested it, and it was a little over a minute for the slowest datasets (ACS block groups), faster than the old manual stats code. I suggested this, as it might be useful to have this information and it is not too slow.

Reply:
I agree that, even if we remove the nulls check, having good statistics for the tables can be useful. Thus, although `check_null_columns` now depends on the `ANALYZE` I would leave the `ANALYZE` outside the function